docs/it/token_endpoint.rst

```diff
@@ -133,31 +127,25 @@ L’ID Token deve essere formato secondo le indicazioni del paragrafo successivo
  }
  
 .. list-table:: 
-   :widths: 20 20 40 20
+   :widths: 20 60 20
    :header-rows: 1
 
    * - **Claim**
-     - **Tipo**
      - **Descrizione**
      - **Obbligatorio**
    * - **access_token**
-     - 
      - L’access token, in formato JWT firmato, consente l’accesso allo UserInfo endpoint per ottenere gli attributi.
      - 
    * - **token_type**
-     - 
      - Tipo di *access token* restituito. Deve essere valorizzato sempre con **Bearer**
      - 
    * - **refresh_token**
-     - 
      - Il *refresh token*, in formato JWT firmato, consente di chiamare nuovamente il Token Endpoint per ottenere un nuovo *access token* e quindi recuperare una `sessione lunga revocabile`_.
      - 
    * - **expires_in**
-     - UNIX Timestamp
-     - Scadenza dell’*access token*, in secondi, secondo le modalità definite dall’Agenzia per l’Italia Digitale.
+     - UNIX Timestamp. Scadenza dell’*access token*, in secondi, secondo le modalità definite dall’Agenzia per l’Italia Digitale.
```

Comment:
- UNIX Timestamp con l’istante di scadenza del JWT, codificato come NumericDate come indicato in
       `RFC7519 - JSON Web Token (JWT) <https://www.rfc-editor.org/rfc/rfc7519.html>`_
       
      rimuovere secondo le modalità definite dall’Agenzia per l’Italia Digitale.

Reply:
**expires_in** non contiene UNIX Timestamp, come **exp**, ma un numero espresso in secondi indicante l'intervallo di validità: 
"_The lifetime in seconds of the access token.  For example, the value "3600" denotes that the access token will expire in one hour from the time the response was generated._" (Rif. [RFC6749, par. 4.2.2](https://datatracker.ietf.org/doc/html/rfc6749#section-4.2.2))

Secondo le modalità definite dall'Agenzia per l'Italia Digitale, perchè nel successivo Avviso SPID n.41 è stato specificato che non deve essere superiore a 300 secondi. (Rif. [Avviso SPID n.41](https://www.agid.gov.it/sites/default/files/repository_files/spid-avviso-n41-integrazione_ll.gg_._openid_connect_in_spid.pdf))

Reply:
oh grazie, my bad!

Contiene solo il delta